# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/test/gtest/TestSurfaceCache.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe image/test/gtest/TestSurfaceCache.cpp
--- a/image/test/gtest/TestSurfaceCache.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/image/test/gtest/TestSurfaceCache.cpp	Wed May 22 22:27:37 2019 +0000
@@ -11,6 +11,7 @@
 #include "ImageFactory.h"
 #include "mozilla/gfx/2D.h"
 #include "mozilla/RefPtr.h"
+#include "mozilla/StaticPrefs.h"
 #include "nsIInputStream.h"
 #include "nsString.h"
 #include "ProgressTracker.h"
@@ -43,7 +44,7 @@
   // Ensures we meet the threshold for FLAG_SYNC_DECODE_IF_FAST to do sync
   // decoding without the implications of FLAG_SYNC_DECODE.
   ASSERT_LT(length,
-            static_cast<uint64_t>(gfxPrefs::ImageMemDecodeBytesAtATime()));
+            static_cast<uint64_t>(StaticPrefs::ImageMemDecodeBytesAtATime()));
 
   // Write the data into the image.
   rv = image->OnImageDataAvailable(nullptr, nullptr, inputStream, 0,